exec/java-exec/src/main/java/org/apache/drill/exec/planner/cost/DrillRelMdSelectivity.java

```diff
@@ -56,6 +56,12 @@
   private static final DrillRelMdSelectivity INSTANCE = new DrillRelMdSelectivity();
   static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(DrillRelMdSelectivity.class);
   public static final RelMetadataProvider SOURCE = ReflectiveRelMetadataProvider.reflectiveSource(BuiltInMethod.SELECTIVITY.method, INSTANCE);
+  /*
+   * For now, we are treating all LIKE predicates to have the same selectivity irrespective of the number or position
+   * of wildcard characters (%). This is no different than the present Drill/Calcite behaviour w.r.t to LIKE predicates.
```

Comment:
There is a difference from the current behavior of Drill/Calcite because the selectivity number has been reduced to 5%.

Reply:
Added the clarification in the comment.